src/main/scala/nl/biopet/utils/tool/ToolDocumentation.scala

```diff
@@ -0,0 +1,163 @@
+package nl.biopet.utils.tool
+
+import java.io.{File, PrintWriter}
+
+import scala.io.Source
+
+trait ToolDocumentation {
+  /** This will return the name of the tool */
+  def toolName: String = this.getClass.getSimpleName.stripSuffix("$")
+
+  // Force description to be written for each tool.
+  def descriptionText: String
+
+  // Force a manual to be written for each tool
+  def manualText: String
+
+  // Force an example to be written for each tool
+  def exampleText: String
+
+  // Force a usage text to be generated for each tool.
+  def usageText: String
+
+  // Universal text for pointing to the documentation.
+  // TODO: Change link.
+  def documentationText: String = s"For documentation and manuals visit our [github.io page](https://biopet.github.io/${toolName})."
+
+  // Universal contact text
+  def contactText: String =
```

Comment:
This is static test or not? Then putting it inside the code is maybe not really needed